docs/02-app/02-api-reference/05-next-config-js/staleTimes.mdx

```diff
@@ -7,7 +7,7 @@ description: Learn how to override the invalidation time of the Client Router Ca
 
 `staleTimes` is an experimental feature that allows configuring the [invalidation period](/docs/app/building-your-application/caching#duration-3) of the client router cache.
 
-This configuration option is available as of v14.2.0-canary.53.
+This configuration option is available as of [v14.2.0-canary.53](https://github.com/vercel/next.js/releases/tag/v14.2.0-canary.53).
```

Comment:
Let's also update this to say 14.2.0 since we got it out in that release rather than the canary